### PR TITLE
adds fantom chains to wagmi config

### LINKED ISSUE #449: general: add Fantom Mainnet and testnet to supported networks
**User Story**
As a project owner, 
I want to be able to create a project on Fantom 
so that I can also apply to a round on Fantom 

**Acceptance Criteria**
GIVEN I am on Grant Hub
WHEN I look at the page 
THEN I see that I can connect to the Fantom network

**Notes:**
- There may be some wallets that are not supported on the Fantom network (i.e. Rainbow)

### PR DESCRIPTION
**Pull request by codenamejason**
Tasks completed:

1. Add Fantom testnet chain
2. Add Fantom mainnet chain

Screen Shot 2022-10-17 at 12 14 46 PM

### CODE REVIEW

**Comment by codenamejason** [comment]
Currently looking into how to disable rainbow wallet option when you select your wallet.

**Comment by codenamejason** [comment]
Removed rainbow wallet

Screen Shot 2022-10-20 at 9 36 45 AM

**Comment by codenamejason** [comment]
Example on Fantom testnet:
Screen Shot 2022-10-20 at 11 48 56 AM

File: client/src/utils/wagmi.ts [hunk 1]
```diff
@@ -26,10 +82,18 @@ export const { chains, provider } = configureChains(chainsAvailable, [
   publicProvider({ priority: 2 }),
 ]);
 
-const { connectors } = getDefaultWallets({
-  appName: "Grant Hub",
-  chains,
-});
+// todo: remove rainbow wallet option
```
Comment (gravityblast) [inline client/src/utils/wagmi.ts]: can we remove this comment if it's already done?

File: client/src/utils/wagmi.ts [hunk 2]
```diff
@@ -10,14 +16,64 @@ const infuraId = process.env.INFURA_ID;
...
+  testnet: false,
+};
+
 if (process.env.REACT_APP_LOCALCHAIN) {
   chainsAvailable.push(chain.hardhat);
 }
 
 if (process.env.REACT_APP_ENV === "production") {
-  chainsAvailable.push(chain.optimism, chain.goerli, chain.optimismKovan);
+  chainsAvailable.push(
```
Comment (gravityblast) [inline client/src/utils/wagmi.ts:65]: shouldn't we put only fantom in production?

**Review by gravityblast — APPROVED** [review]
LGTM!! just added a couple of comments

File: client/src/utils/wagmi.ts (hunk 2 above)
Comment (codenamejason) [reply client/src/utils/wagmi.ts:65]: this was per @michellema1208 request

**Pull request merged**